LICENSE-APACHE

```diff
@@ -186,7 +186,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2010-2017 The Rust Project Developers
```

Comment:
This is explaining how to apply the license to your own work, it's not a real copyright statement